client/my-sites/plan-features/header.jsx

```diff
@@ -76,18 +69,14 @@ export class PlanFeaturesHeader extends Component {
 			translate,
 		} = this.props;
 
-		const headerClasses = classNames( 'plan-features__header', getPlanClass( planType ), {
-			'is-p2-plus': planType === PLAN_P2_PLUS,
```

Comment:
Should we also remove the CSS styles for this class?
https://github.com/Automattic/wp-calypso/blob/0c16202f7a022efff85d1097bcbd4818b7df869b/client/my-sites/plan-features/style.scss#L315-L324

Reply:
Yes you're totally right. Thanks for catching this @aneeshd16  😎

Reply:
https://github.com/Automattic/wp-calypso/pull/83320/commits/4a3248ad4da05de96e6ccc8aa1931ff13371b7ca